### ISSUE TITLE
Add option for text box width

### ISSUE THREAD

**Issue by shaneporter** [att/rcloud.rcap#185]
This is addressed in a more generic way by #103.

**Comment by shaneporter** [comment]
As a workaround, this can be specified in a theme or `rcap_designer.css` asset;

````css
.grid-stack-item-content,
.grid-stack-item-content .rcap-form,
.grid-stack-item-content .rcap-form input[type="text"] {
    width: 100%;
}
````

**Comment by shaneporter** [comment]
Fixed with PR #225.
